contracts/abstracts/Controlable.sol

```diff
@@ -67,7 +77,7 @@ abstract contract Controlable is AccessControlUpgradeable {
     success = true;
   }
 
-  function _blacklistUser(address userAddress, bool set) internal returns (bool success) {
+  function _blacklistUser(address userAddress, bool set) internal onlyModerator returns (bool success) {
```

Comment:
since you already added the modifier on the external function, we don't need to add it to the internal function